### PR TITLE
Added documentation for Operations

### PR DESCRIPTION
**Pull request by ninlar**
Architecture has requested that Operations document our APIs. Previously these APIs were undocumented and the consumer was restricted to the OCS portal.

### CODE REVIEW

File: Documentation/Operations/Operations_Billing Cycles.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+# Billing Cycles
+
+APIs related to querying Billing Cycles
+
+
+***
+
+## Authentication
+
+All endpoints referenced in this documentation require authenticated access. Authorization header must be set to the access token you retrieve after a successful authentication request.
```
Comment (osimarkrogers) [inline Documentation/Operations/Operations_Billing Cycles.md]: "retrieve" should be "retrieved"

File: Documentation/Operations/Operations_Billing Cycles.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+`Authorization: Bearer <token>`
+
+Requests made without an access token or an invalid/expired token will fail with a 401 Unauthorized response.
+Requests made with an access token which does not have the correct permissions (see security subsection on every endpoint) will fail with a 403 Forbidden.
+Read [here](https://github.com/osisoft/OSI-Samples-OCS/tree/master/basic_samples/Authentication) on how to authenticate against OCS with the various clients and receive an access token in response.
```
Comment (osimarkrogers) [inline Documentation/Operations/Operations_Billing Cycles.md]: Replace "read [here]" with "read the [OCS Authentication documentation]"

File: Documentation/Operations/Operations_Billing Cycles.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+## Error Handling
+
+All responses will have an error message in the body. The exceptions are 200 responses and the 401 Unauthorized response. The error message will look as follows:
```
Comment (osimarkrogers) [inline Documentation/Operations/Operations_Billing Cycles.md]: Should be "The error message will appear as follows:"

File: Documentation/Operations/Operations_Logs.md [hunk 4]
```diff
@@ -0,0 +1,305 @@
...
+# Logs
+
+APIs related to querying logs.
+
+
+***
+
+## Authentication
+
+All endpoints referenced in this documentation require authenticated access. Authorization header must be set to the access token you retrieve after a successful authentication request.
```
Comment (osimarkrogers) [inline Documentation/Operations/Operations_Logs.md]: "retrieved"

File: Documentation/Operations/Operations_Logs.md [hunk 5]
```diff
@@ -0,0 +1,305 @@
...
+
+`Authorization: Bearer <token>`
+
+Requests made without an access token or an invalid/expired token will fail with a 401 Unauthorized response.
+Requests made with an access token which does not have the correct permissions (see security subsection on every endpoint) will fail with a 403 Forbidden.
+Read [here](https://github.com/osisoft/OSI-Samples-OCS/tree/master/basic_samples/Authentication) on how to authenticate against OCS with the various clients and receive an access token in response.
```
Comment (osimarkrogers) [inline Documentation/Operations/Operations_Logs.md]: Change to "read [OCS Authentication documentation]"

File: Documentation/Operations/Operations_Resource Usage Data.md [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+# Resource Usage Data
+
+APIs related to querying Resource Usage Data
+
+
+***
+
+## Authentication
+
+All endpoints referenced in this documentation require authenticated access. Authorization header must be set to the access token you retrieve after a successful authentication request.
```
Comment (osimarkrogers) [inline Documentation/Operations/Operations_Resource Usage Data.md]: "retrieved"

File: Documentation/Operations/Operations_Resource Usage Data.md [hunk 7]
```diff
@@ -0,0 +1,95 @@
...
+
+`Authorization: Bearer <token>`
+
+Requests made without an access token or an invalid/expired token will fail with a 401 Unauthorized response.
+Requests made with an access token which does not have the correct permissions (see security subsection on every endpoint) will fail with a 403 Forbidden.
+Read [here](https://github.com/osisoft/OSI-Samples-OCS/tree/master/basic_samples/Authentication) on how to authenticate against OCS with the various clients and receive an access token in response.
```
Comment (osimarkrogers) [inline Documentation/Operations/Operations_Resource Usage Data.md]: "Read the [OCS Authentication documentation]"

File: Documentation/Operations/Operations_Usage Data.md [hunk 8]
```diff
@@ -0,0 +1,250 @@
...
+# Usage Data
+
+APIs related to querying Usage data.
+
+
+***
+
+## Authentication
+
+All endpoints referenced in this documentation require authenticated access. Authorization header must be set to the access token you retrieve after a successful authentication request.
```
Comment (osimarkrogers) [inline Documentation/Operations/Operations_Usage Data.md]: "retrieved"

File: Documentation/Operations/Operations_Usage Data.md [hunk 9]
```diff
@@ -0,0 +1,250 @@
...
+
+`Authorization: Bearer <token>`
+
+Requests made without an access token or an invalid/expired token will fail with a 401 Unauthorized response.
+Requests made with an access token which does not have the correct permissions (see security subsection on every endpoint) will fail with a 403 Forbidden.
+Read [here](https://github.com/osisoft/OSI-Samples-OCS/tree/master/basic_samples/Authentication) on how to authenticate against OCS with the various clients and receive an access token in response.
```
Comment (osimarkrogers) [inline Documentation/Operations/Operations_Usage Data.md]: "Read the [OCS Authentication documentation]"

Comment (ninlar) [reply Documentation/Operations/Operations_Usage Data.md]: Resolved

File: Documentation/Operations/Operations_Usage Data.md (hunk 8 above)
Comment (ninlar) [reply Documentation/Operations/Operations_Usage Data.md]: Resolved

File: Documentation/Operations/Operations_Resource Usage Data.md (hunk 7 above)
Comment (ninlar) [reply Documentation/Operations/Operations_Resource Usage Data.md]: Resolved

File: Documentation/Operations/Operations_Resource Usage Data.md (hunk 6 above)
Comment (ninlar) [reply Documentation/Operations/Operations_Resource Usage Data.md]: Resolved

Comment (ninlar) [reply Documentation/Operations/Operations_Resource Usage Data.md]: Resolved

File: Documentation/Operations/Operations_Logs.md (hunk 5 above)
Comment (ninlar) [reply Documentation/Operations/Operations_Logs.md]: Resolved

File: Documentation/Operations/Operations_Logs.md (hunk 4 above)
Comment (ninlar) [reply Documentation/Operations/Operations_Logs.md]: Resolved

File: Documentation/Operations/Operations_Billing Cycles.md (hunk 3 above)
Comment (ninlar) [reply Documentation/Operations/Operations_Billing Cycles.md]: Resolved

File: Documentation/Operations/Operations_Billing Cycles.md (hunk 2 above)
Comment (ninlar) [reply Documentation/Operations/Operations_Billing Cycles.md]: Resolved

File: Documentation/Operations/Operations_Billing Cycles.md (hunk 1 above)
Comment (ninlar) [reply Documentation/Operations/Operations_Billing Cycles.md]: Resolved

**Pull request merged**